src/components/Button/Button.stories.tsx

```diff
@@ -29,7 +27,7 @@ export default {
   argTypes: {
     onClick: { action: 'clicked' },
     variant: {
-      options: ['primary', 'secondary', 'ghost', 'link', 'unstyled'],
```

Comment:
We're no longer being haunted by the ghost button? This is definitely a better name...